pyuploadcare/api/api.py

```diff
@@ -510,8 +510,11 @@ def _get_request_data(
         file_uuid: Union[UUID, str],
         params: Optional[AddonExecutionParams] = None,
     ) -> dict:
+        cleaned_params = {}
+        if params:
+            cleaned_params = params.dict(exclude_unset=True, exclude_none=True)
```

Comment:
Fixed a bug here by filtering out `None` values. If we don't, server will return errors:

```json
{
  "scale": [
    "This field may not be null."
  ],
  "type": [
    "This field may not be null."
  ],
  "roi": [
    "This field may not be null."
  ],
  "position": [
    "This field may not be null."
  ]
}
```